src/main/java/org/jboss/tools/intellij/openshift/utils/odo/OdoCli.java

```diff
@@ -153,6 +155,7 @@ public OdoCli(com.intellij.openapi.project.Project project, String command) {
         String current = ConfigHelper.getCurrentContextName();
         Config config = Config.autoConfigure(current);
         this.client = new KubernetesClientBuilder().withConfig(config).withHttpClientBuilderConsumer(builder -> setSslContext(builder, config)).build();
+        this.openshiftClient = toOpenShiftClient(client);
```

Comment:
wonder if using the client as param makes sense ? `toOpenShiftClient` method is used only at this place now.

Reply:
I would keep it. Method signature lays bare that a client is adapted, something you'd only discover reading the method body if the instance variable was used. Furthermore it makes it immediately clear that one needs to initialize the kubernetes client first.